resources/sass/_base.scss

```diff
@@ -1,47 +1,47 @@
 // Variables for themes to override
-$spacing: 4 !default;
-$color_size: 16 !default;
+:root {
+  --spacing: 4px;
+  --color-size: 16px;
 
-$backdrop-color: rgba(#fff, 0.9) !default;
-$main-background: #e0eaef !default;
-$secondary-background: #c1d4df !default;
+  --backdrop-color: rgba(255, 255, 255, 0.9);
```

Comment:
The technical reason for the `--backdrop-color` change is that it needs to be overridable from other themes.

The reason behind changing `rgba(#fff, 0.9)` to `rgba(255, 255, 255, 0.9)` is because the former doesn't appear to be valid CSS3 color, while the latter is.  We could still use Sass-style colors, but it needs to be transpiled into regular CSS for it to work and I couldn't figure it out in the spur of the moment.

Are we heading towards a future without Sass? No, it doesn't make sense to stop using all the nice benefits of Sass, such as nested declarations. The intention is to only support using native CSS variables, so that they can be overridden in run time (rather than build time) by other themes, depending on the situation (e.g. using `media` queries), which can't be done using Sass-only variables.

Reply:
Ugh. Got it working but the syntax sucks:

```scss
--backdrop-color: #{rgba(#FFF, 0.9)};
```

Reply:
Excellent, and I agree the syntax is less than visually appealing, but where we _need_ to make use of Sass utils, it's acceptable.

I would say that after applying a "color refresh" to make it look better, some of these Sass definitions wouldn't be used as we will just apply an absolute CSS color.